api/src/modules/scenario-interventions/scenario-intervention.entity.ts

```diff
@@ -112,25 +127,51 @@ export class ScenarioIntervention extends TimestampedBaseEntity {
   /**
    * Relationships with other entities - list of "new" relationships
    */
-  @ManyToMany(() => Material)
-  @JoinTable()
-  newMaterials?: Material[];
+  @ManyToOne(() => Material)
+  @ApiPropertyOptional({ type: () => Material })
+  newMaterial?: Material;
 
-  @ManyToMany(() => BusinessUnit)
-  @JoinTable()
-  newBusinessUnits?: BusinessUnit[];
+  @ManyToOne(() => BusinessUnit)
+  @ApiPropertyOptional()
+  newBusinessUnit?: BusinessUnit;
 
-  @ManyToMany(() => Supplier)
-  @JoinTable()
-  newSuppliers?: Supplier[];
+  @ManyToOne(() => Supplier)
+  @ApiPropertyOptional()
+  newT1Supplier?: Supplier;
 
-  @ManyToMany(() => AdminRegion)
-  @JoinTable()
-  newAdminRegion?: AdminRegion[];
+  @ManyToOne(() => Supplier)
+  @ApiPropertyOptional()
+  newProducer?: Supplier;
 
-  @ManyToMany(() => SourcingLocation)
-  @JoinTable()
-  newSourcingLocation?: SourcingLocation[];
+  @ManyToOne(() => AdminRegion)
+  @ApiPropertyOptional()
+  newAdminRegion?: AdminRegion;
+
+  @OneToMany(
+    () => SourcingLocation,
+    (sourcingLocation) => sourcingLocation.scenarioIntervention,
```

Comment:
Doesn't your IDE complain about:
`ESLint: Expected sourcingLocation to have a type annotation.(@typescript-eslint/typedef)`
here? 
If it doesn't maybe worth your linter settings 😄